### DISCUSSION TITLE
Beta tag release notes

### DISCUSSION THREAD

**Discussion by derekcentrico** [haugene/docker-transmission-openvpn#2544] [General]
I'm not seeing release note for the beta tag.  Curious about whether I want to hop onto it for Transmission 4 or just stay chill where I am until a stable release.  Seeing all that's baked into it would be helpful.

**Comment by pkishino** [comment]
Feel free to check transmissions own release notes for it and/or the commit history

**Reply by derekcentrico** [reply]
So does that mean that beta is solely transmission 4.x and not anything else merged in?  I saw a lot of dev to beta merging at a quick glance so I figured wireguard work among other things may be there.

**Reply by cpainchaud** [reply]
> Feel free to check transmissions own release notes for it and/or the commit history

Hi  @pkishino ,
is there a way/place to report beta specific issues? For example right now it seems to write nothing in transmission.log

**Reply by pkishino** [reply]
anything related to transmission itself should be reported to the official transmission repo

**Reply by cpainchaud** [reply]
> anything related to transmission itself should be reported to the official transmission repo

Right but it seems to be related to this image and environment, I mean transmission's logging seems to work when I use it outside of docker. Do you see what I mean?

**Comment by pkishino** [comment]
This image doesn’t “do” anything with transmission except package it.. if
it doesn’t log then most likely your permissions are wrong for wherever it
is trying to write to or you have made some mistake in the container
parameters.
You should check your container logs.

**Reply by cpainchaud** [reply]
I will double check. I think that they may have changed their default logging levels because if I modify this project's launcher environment vars I can get it log things again
